CMakeLists.txt

```diff
@@ -153,6 +153,46 @@ if(ITKPythonPackage_SUPERBUILD)
   message(STATUS "SuperBuild - Searching for python[OK]")
   message(STATUS "SuperBuild -   DOXYGEN_EXECUTABLE: ${DOXYGEN_EXECUTABLE}")
 
+  # CMake configuration variables to pass to ITK's build
+  set(ep_itk_cmake_cache_args "")
+  foreach(var ITK_WRAP_DOC
```

Comment:
`ITK_WRAP_DOC` should be indented like the other variables

Sorting alphabetically also helps.

Reply:
Done